src/bolero/bgp.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2025 Hedgehog
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
SPDX header should always come first[^1].

The inversion present in this repo is a mistake and should be fixed up eventually. Please consider swapping the order of the lines for the new headers you add in this PR.

As a side note, CNCF guidelines say that years are not required in copyright[^2]. Better remain consistent in the repo, though.

[^1]: I think I saw that in some SPDX docs, I can at least see it in [kernel's docs](https://docs.kernel.org/process/license-rules.html#license-identifier-syntax)
[^2]: https://github.com/cncf/foundation/blob/main/copyright-notices.md#copyright-notices

Reply:
Fixed in files I added.

Reply:
The actual generator is the addlicense project by Google so I don't think it needs a fix.  I don't mind having things in the SPDX-first order in my files but I don't think spec requires it, see here: https://spdx.github.io/spdx-spec/v2.3/using-SPDX-short-identifiers-in-source-files/

Reply:
Apparently, there's no consensus: https://github.com/google/addlicense/issues/131

So fine to keep as it is in this repo, I suppose :shrug: 
Sorry for the distraction